system/integrations/index.rst

```diff
@@ -40,6 +40,7 @@ Integrations for authentication and customers
    exchange
    clearbit
    smime/index
+   pgp/index
```

Comment:
Could we please reorder the list of these integrations alphabetically? It's becoming too long to reliable identify topic of interest. Feel free to apply the same reasoning to other integrations block if you wish.